Lib/asyncio/runners.py

```diff
@@ -45,6 +45,7 @@ def run(main, *, debug=False):
         try:
             _cancel_all_tasks(loop)
             loop.run_until_complete(loop.shutdown_asyncgens())
+            loop.run_until_complete(loop.shutdown_default_executor())
```

Comment:
Since this is now a coroutine, it should be called with ``loop.run_until_complete()`` instead of by itself as it was previously.